src/ResourceManager/Resources/Commands.Resources.Test/Models.ResourceGroups/ResourceClientTests.cs

```diff
@@ -1391,13 +1391,13 @@ public void CancelsActiveDeployment()
         public void SerializeHashtableProperlyHandlesAllDataTypes()
         {
             Hashtable hashtable = new Hashtable();
-            var password = "pass";
+            var pass = "pass";
             hashtable.Add("key1", "string");
```

Comment:
The right way to avoid false positives matches is to use [inline-suppressions](https://microsoft.sharepoint.com/teams/CESecEngineering/CredScan/CredScan%20Wiki/Suppression%20Usage%20Examples.aspx).

Reply:
This one should be fine. It's a random variable, and doesn't have to be named "password".